subsys/bluetooth/controller/Kconfig.df

```diff
@@ -0,0 +1,120 @@
+# Zephyr Bluetooth Controller configuration options
```

Comment:
You are defining a huge array of Kconfig options that you are not using. Could we please introduce those when the need arises, including a use for them? I haven't checked thoroughly but I have a feeling you are not using them all yet

Reply:
Ok, I'll remove those not used at the time.